eth/catalyst/api.go

```diff
@@ -133,9 +133,9 @@ type blockExecutionEnv struct {
 }
 
 func (env *blockExecutionEnv) commitTransaction(tx *types.Transaction, coinbase common.Address) error {
-	vmconfig := *env.chain.GetVMConfig()
+	vmConfig := env.chain.GetVMConfig()
```

Comment:
What happened here? It previously did the copy first, but you changed it so it does the copy later, where it's much more difficult to spot (out on the far right). I think it was better as it was before.

Reply:
I made a change where I passed the config directly not by pointer. When I undid it, I must've made this change